### PR TITLE
docs: add in-code docs for the get command (#AS-1398)

### PR DESCRIPTION
**Pull request by mlikasam-askui**

### CODE REVIEW

File: packages/askui-nodejs/src/execution/dsl.ts [hunk 1]
```diff
@@ -3713,7 +3713,17 @@ export class FluentFiltersOrRelationsGetter extends FluentFiltersGetter {
...
+   * A detected element has the following properties:
+   * - `name` of the element
+   * - `text` content of element
+   * - `colors` of element
+   * - `bndbox`: location of element described with coordinates of a bounding box
+   * **Examples:**
+   * ```typescript
+   * const icon = await aui.get().icon().withText('care left').exec();
+   * ```
+   *
```
Comment (adi-wan-askui) [inline packages/askui-nodejs/src/execution/dsl.ts:3726]: Rm empty line

File: packages/askui-nodejs/src/execution/dsl.ts [hunk 2]
```diff
@@ -3713,7 +3713,17 @@ export class FluentFiltersOrRelationsGetter extends FluentFiltersGetter {
...
-   * Returns the filtered element
+   * Returns an array with all filtered elements.
+   * A detected element has the following properties:
+   * - `name` of the element
+   * - `text` content of element
+   * - `colors` of element
+   * - `bndbox`: location of element described with coordinates of a bounding box
+   * **Examples:**
+   * ```typescript
+   * const icon = await aui.get().icon().withText('care left').exec();
```
Comment (adi-wan-askui) [inline packages/askui-nodejs/src/execution/dsl.ts:3724]: Can you add an example of what is returned ;)

```suggestion
   * const icon = await aui.get().icon().withText('care left').exec();
   * console.log(icon)
   * // [{ name: 'icon', text: '...', colors: [...], ... }, ...]
```

(add the details to what is logged, maybe a multiline comment would be better)

File: packages/askui-nodejs/src/execution/dsl.ts [hunk 3]
```diff
@@ -3724,7 +3734,16 @@ export abstract class Getter extends FluentCommand {
...
-   * Returns all detected elements
+   * Returns an array with all detected elements.
+   * A detected element has the following properties:
+   * - `name` of the element
+   * - `text` content of element
+   * - `colors` of element
+   * - `bndbox`: location of element described with coordinates of a bounding box
+   * **Examples:**
+   * ```typescript
+   * const detectedElements = await aui.getAll().exec()
```
Comment (adi-wan-askui) [inline packages/askui-nodejs/src/execution/dsl.ts:3745]: Same here, example of what is returned

**Review by adi-wan-askui — CHANGES_REQUESTED** [review]
There it is! Like :)

File: docs/docs/api/06-Getters/getall.md [hunk 4]
```diff
@@ -10,39 +9,35 @@ A detected element has the following properties:
...
-      ...
-]
+```text 
+ console output: [
+  DetectedElement {
+     name: 'TEXT',
+     text: 'Sign In',
+     colors: [ 'black', 'gray', 'gray' ],
+     bndbox: BoundingBox {
+     xmin: 1128.2720982142857,
```
Comment (JohannesDienst-askui) [inline docs/docs/api/06-Getters/getall.md:24]: Please correct the indentation 😉

File: docs/docs/api/06-Getters/getall.md [hunk 5]
```diff
@@ -10,39 +9,35 @@ A detected element has the following properties:
...
+     ymin: 160.21332310267857,
+     xmax: 1178.8204241071428,
+     ymax: 180.83512834821428
+     },
+  DetectedElement {
+     name: 'ICON',
+     text: 'search',
+     colors: [ 'black', 'red', 'gray' ],
+     bndbox: BoundingBox {
+     xmin: 250.8204241071428,
```
Comment (JohannesDienst-askui) [inline docs/docs/api/06-Getters/getall.md:34]: Please correct the indentation 😉

**Review by JohannesDienst-askui — CHANGES_REQUESTED** [review]

**Review by JohannesDienst-askui — APPROVED** [review]

**Review by adi-wan-askui — APPROVED** [review]

**Pull request merged**